### PR TITLE
Show span ID and parent ID in span detail

### LINKED ISSUE #2921: Lens UI missing raw span and trace ids
Lens UI has lost the ability to show raw spanId, traceId, and parentSpanId in the recent UI overhaul. Reproduced in zipkin 2.19.1.

See #2507 and #1300 for when this was added the last two times

### PR DESCRIPTION
**Pull request by tacigar**
resolves #2921 

スクリーンショット 2019-11-29 17 41 13

cc @lambcode

### CODE REVIEW

File: zipkin-lens/src/components/TracePage/SpanDetail/SpanDetail.jsx [hunk 1]
```diff
@@ -24,51 +24,114 @@ import { detailedSpanPropTypes } from '../../../prop-types';
...
+  const classes = useStyles();
+
+  const spanIds = (
+    <Box className={classes.spanIds}>
+      {
+        [
+          { label: 'Span ID', value: span.spanId },
+          { label: 'Parent ID', value: span.parentId },
+        ].map((entry) => {
+          if (!entry.value) {
```
Comment (anuraaga) [inline zipkin-lens/src/components/TracePage/SpanDetail/SpanDetail.jsx]: What does it look like when there's no parent ID? Maybe rendering with `-` is better than skipping

Comment (tacigar) [reply zipkin-lens/src/components/TracePage/SpanDetail/SpanDetail.jsx]: I see.

**Review by anuraaga — APPROVED** [review]

**Comment by lambcode** [comment]
Looks great. Thanks!

**Comment by jcchavezs** [comment]
I am not sure about the `--` for when lack of `parent`. Maybe `None` is better?

**Comment by tacigar** [comment]
I see :+1:

**Comment by tacigar** [comment]
None

**Pull request merged**